easypost/__init__.py

```diff
@@ -77,7 +77,8 @@ def convert_to_easypost_object(response, api_key, parent=None, name=None):
         'PickupRate': PickupRate,
         'PostageLabel': PostageLabel,
         'CarrierAccount': CarrierAccount,
-        'User': User
+        'User': User,
+        'Report': Report,
```

Comment:
You may need to add a similar mapping to the `prefixes` below (I can't comment on that line, stats on 84)